### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'value'

### ISSUE THREAD

**Issue by thijsdejong** [iMicknl/ha-tahoma#646]
### Did you read the instructions?

- [X] I have read the [README](https://github.com/iMicknl/ha-tahoma/blob/master/README.md), including the [advanced debugging](https://github.com/iMicknl/ha-tahoma/blob/master/README.md#advanced) section.

### The problem

When running an action on any cover I get the following error pop up:

`AttributeError: 'NoneType' object has no attribute 'value'`

Has been happening since a week or 2 or so after an update of ha-tahoma. Can't find which version unfortunately.

### What version of this integration (ha-tahoma) has the issue?

2.9.2

### What version of Home Assistant Core has the issue?

core-2021.11.0

### Which gateway / hub do you use?

Somfy TaHoma

### Device model

PositionableRollerShutterWithLowSpeedManagement

### Home Assistant log

```shell
2021-11-04 16:02:25 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547782063440] 'NoneType' object has no attribute 'value'
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 185, in handle_call_service
await hass.services.async_call(
File "/usr/src/homeassistant/homeassistant/core.py", line 1495, in async_call
task.result()
File "/usr/src/homeassistant/homeassistant/core.py", line 1530, in _execute_service
await handler.job.target(service_call)
File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 213, in handle_service
await self.hass.helpers.service.entity_service_call(
File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 667, in entity_service_call
future.result() # pop exception if have
File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 863, in async_request_call
await coro
File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 704, in _handle_entity_call
await result
File "/config/custom_components/tahoma/cover_devices/vertical_cover.py", line 128, in async_close_cover
await self.executor.async_execute_command(
File "/config/custom_components/tahoma/executor.py", line 83, in async_execute_command
await self.coordinator.async_refresh()
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 165, in async_refresh
await self._async_refresh(log_failures=True)
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 265, in _async_refresh
update_callback()
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 325, in _handle_coordinator_update
self.async_write_ha_state()
File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 486, in async_write_ha_state
self._async_write_ha_state()
File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 519, in _async_write_ha_state
state = self._stringify_state()
File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 492, in _stringify_state
if (state := self.state) is None:
File "/usr/src/homeassistant/homeassistant/components/number/__init__.py", line 139, in state
return self.value
File "/config/custom_components/tahoma/number.py", line 65, in value
return self.device.states.get(self.entity_description.key).value
AttributeError: 'NoneType' object has no attribute 'value'
```

**Comment by vlebourl** [comment]
Hi, 
Thanks for your report. Version 2.9.3 was released some time ago. Could you update and report whether the issue persists?

**Comment by mstuij** [comment]
I had the same problem and I can confirm that it is solved with the release of today (2.9.3)
